packages/marketplace/src/components/ui/analytics/detailed/__test__/detailed-tab.test.tsx

```diff
@@ -91,6 +92,13 @@ describe('OverviewPage', () => {
           pageSize: 9999,
         }),
       )
+      expect(spyDispatch).toBeCalledWith(
+        httpTrafficPerDayRequestData({
+          applicationId: ['09043eb8-9e5e-4650-b7f1-f0cb62699027', '261da083-cee2-4f5c-a18f-8f9375f1f5af'],
```

Comment:
Why do we expect `httpTrafficPerDayRequestData` action to be dispatched when `handleFetchAppUsageStatsDataUseCallback` function runs?
`httpTrafficPerDayRequestData` action should be dispached when `handleFetchHttpTrafficPerDayDataUseCallback` runs instead